core/src/layers/logging.rs

```diff
@@ -27,7 +27,7 @@ use futures::TryFutureExt;
 use log::debug;
 use log::log;
 use log::trace;
-use log::Level;
+pub use log::Level;
```

Comment:
This type should not be public from `opendal`.

Reply:
Why? Or I think providing a `Level` wrapper in opendal would be good. The user just want to configure the logging level, but they have to add a new crate.

Reply:
Oh, it's a mistake of our API design. We should not expose external crate types directly.

Would you like to creatse a new issue for this? I prefer to change the API into `with_error_level(lvl: Option<&str>)`.

Reply:
@Xuanwo Cool, that would be better.